src/benchmarks/beluga_vs_nav2/Earthfile

```diff
@@ -34,6 +34,7 @@ devel:
         apt clean && rm -rf /var/lib/apt/lists/*
     RUN pip install linuxdoc sphinxcontrib.datatemplates sphinxcontrib-repl
     DO os+ADDUSER --user=${user} --uid=${uid} --gid=${gid} --workdir=/workspace
+    DO os+FIXUSER --user=${user} --workdir=/workspace/src
```

Comment:
@marcoshuck why does this still take `--user` and `--workdir`?

Reply:
Because of my lack of attention to detail, thank you! Just removed it.